scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -293,25 +293,49 @@ def process_int32(self, data):
             signum = True
             buf = self._read_bytes(4)
             data.append(int.from_bytes(buf, byteorder=self.config['byteorder'],
-                                    signed=signum))
+                                       signed=signum))
 
         def process_uint32(self, data):
             signum = False
             buf = self._read_bytes(4)
             data.append(int.from_bytes(buf, byteorder=self.config['byteorder'],
-                                    signed=signum))
+                                       signed=signum))
+
+        def process_int16(self, data):
+            signum = True
```

Comment:
We may even ignore defining `signum` as a variable here and simply pass `True`/`False` to the function - we pass it as named function argument,, so it's clear what it means.